operators/reg_patch_constructor.py

```diff
@@ -47,17 +47,30 @@ def get_neighbor_verts(cls, vert) -> list:
         get_vert_order = [1, 0, 3, 6, 7, 8, 5, 2]
         nb_verts = Halfedge.get_verts_repeat_n_times(he, commands, 4, get_vert_order, 9)
         nb_verts[4] = vert
-
         return nb_verts
 
+    @classmethod
```

Comment:
Could you reduce the duplicated code of "get_bezier_patch" and "get_patch"?
e.g. get_patch calls get_bezier_patch